docs/conf.py

```diff
@@ -31,6 +31,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'sphinx.ext.doctest',
```

Comment:
It might already be happening depending on how we're running our doctests (I'm not too sure anymore).